### PR TITLE
fixing some errors in footer component

### PR DESCRIPTION
**Pull request by Entkenntnis**

### CODE REVIEW

**Comment by Entkenntnis** [comment]
local build is working, why is this test failing?

File: packages/public/frontend/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "lint": "yarn tsc --noemit",
     "serverless": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3000 node serverless/serverless.development.js",
     "start": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3000 next",
+    "start-dev": "cross-env NEXT_ASSET_PREFIX=http://localhost:3000 ASSET_PREFIX=http://localhost:3001 next",
```
Comment (inyono) [inline packages/public/frontend/package.json]: Why do we need to commands (`start` and `start-dev`) here?

File: packages/public/frontend/src/assets.ts [hunk 2]
```diff
@@ -1,4 +1,8 @@
-const assetPrefix = process.env.ASSET_PREFIX || ''
+let assetPrefix = process.env.ASSET_PREFIX || ''
+
+export function setAssetPrefix(path) {
```
Comment (inyono) [inline packages/public/frontend/src/assets.ts]: Why is this needed? Shouldn't this always be `process.env.ASSET_PREFIX`?

File: packages/public/frontend/package.json (hunk 1 above)
Comment (Entkenntnis) [reply packages/public/frontend/package.json]: The start-dev command uses the frontend-assets service in docker setup, so its useful for me, maybe change the name to something like start-with-asset-server ...

File: packages/public/frontend/src/assets.ts (hunk 2 above)
Comment (Entkenntnis) [reply packages/public/frontend/src/assets.ts]: Unfortunately not when this code is run on the client, so we need to set it explicitly

File: packages/public/frontend/package.json (hunk 1 above)
Comment (inyono) [reply packages/public/frontend/package.json]: Mh... But isn't `next` already serving the assets in development anyways?

File: packages/public/frontend/src/assets.ts (hunk 2 above)
Comment (inyono) [reply packages/public/frontend/src/assets.ts]: I see. Then why is this different compared to the other usages of the asset prefix? So far, we pass those via props, don't we?

Furthermore: it seems like there are already a couple usages of the assetPrefix in the project. What do you think about an `assetPrefixContext` that we pretty much set once at the root? And every component that needs it (doesn't matter how deep in the tree), uses that context.

Comment (Entkenntnis) [reply packages/public/frontend/src/assets.ts]: I haven't decided yet which way to go, a context would be great and maybe it can we implemented at nextjs level, so pages don't need to care about it ... I feel this would fit into a separate PR

Comment (inyono) [reply packages/public/frontend/src/assets.ts]: That's fine by me. But the current usages should be done in a similar way, right?

File: packages/public/frontend/package.json (hunk 1 above)
Comment (Entkenntnis) [reply packages/public/frontend/package.json]: sadly this leads to CORS issues

File: packages/public/frontend/src/assets.ts (hunk 2 above)
Comment (Entkenntnis) [reply packages/public/frontend/src/assets.ts]: done
